lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/auth/DBOAuthenticationDAOImpl.java

```diff
@@ -62,38 +66,31 @@
 	public static final Long SESSION_EXPIRATION_TIME = 1000 * 60 * 60 * 24L;
 	public static final long HALF_SESSION_EXPIRATION = SESSION_EXPIRATION_TIME/2;
 	
-	public static final String SELECT_VALIDATED_ON_FROM_SESSION_TOKEN = 
-			"SELECT "+COL_SESSION_TOKEN_VALIDATED_ON+
-			" FROM "+TABLE_SESSION_TOKEN+
-			" WHERE "+COL_SESSION_TOKEN_PRINCIPAL_ID+" = ?";
+	public static final String SELECT_AUTHENTICATED_ON_FOR_PRINCIPAL_ID = 
+			"SELECT "+COL_AUTHENTICATED_ON_AUTHENTICATED_ON+
+			" FROM "+TABLE_AUTHENTICATED_ON+
+			" WHERE "+COL_AUTHENTICATED_ON_PRINCIPAL_ID+" = ?";
 	
 	private static final String SELECT_COUNT_BY_EMAIL_AND_PASSWORD =
 			"SELECT COUNT(*)"+
 			" FROM "+TABLE_CREDENTIAL+", "+TABLE_USER_GROUP+
 			" WHERE "+COL_CREDENTIAL_PRINCIPAL_ID+"="+COL_USER_GROUP_ID+
-				" AND "+COL_USER_GROUP_ID+"= ?"+
-				" AND "+COL_CREDENTIAL_PASS_HASH+"=?";
+			" AND "+COL_USER_GROUP_ID+"= ?"+
+			" AND "+COL_CREDENTIAL_PASS_HASH+"=?";
 	
-	private static final String UPDATE_VALIDATION_TIME = 
-			"UPDATE "+TABLE_SESSION_TOKEN+
-			" SET "+COL_SESSION_TOKEN_VALIDATED_ON+"= ?"+
-			" WHERE "+COL_SESSION_TOKEN_PRINCIPAL_ID+"= ?";
-
-	private static final String IF_VALID_SUFFIX =
-			" AND "+COL_SESSION_TOKEN_VALIDATED_ON+"> ?";
+	private static final String UPDATE_AUTHENTICATED_ON = 
+			"UPDATE "+TABLE_AUTHENTICATED_ON+
+			" SET "+COL_AUTHENTICATED_ON_AUTHENTICATED_ON+"= ?, "+COL_AUTHENTICATED_ON_ETAG+"=?"+
```

Comment:
We might want to use COL_AUTHENTICATED_ON_ETAG  + " = UUID()" instead of generating it in the app.